instrumentation/internal/internal-slf4j/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/internal/slf4j/Slf4jInstrumentationModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.internal.slf4j;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class Slf4jInstrumentationModule extends InstrumentationModule {
+
+  public Slf4jInstrumentationModule() {
+    super("internal-slf4j");
+  }
+
+  @Override
+  public boolean defaultEnabled(ConfigProperties config) {
+    // only enable this instrumentation if the application logger is enabled
+    return config.getString("otel.javaagent.logging", "simple").equalsIgnoreCase("application");
```

Comment:
It looks like this was around before this PR, but I don't see mention of it in any readme/notes/docs (and I also search the docs repo). Can we please get this added somewhere or an issue created to do that work?

Reply:
Yeah, I was planning to add some docs once this PR is accepted. Will create an issue when this is merged.